ads/_config.js

```diff
@@ -110,6 +110,7 @@ export const adConfig = {
     ],
     renderStartImplemented: true,
   },
+  adverticum: {},
```

Comment:
nit: empty line above.
Also I see you have `renderstartImplemented` in the other PR. If you implemented `renderStart` that value is needed here. If not, I would recommend implementing `renderStart` API for better UX.